src/source/core/Tasks.bs

```diff
@@ -112,6 +112,22 @@ namespace mc.tasks
       mc.tasks.observeNodeField(task, fieldName, callback, sendMode, true, scope)
     end if
 
+    if asBoolean(globalObj.isTrackingTaskUsage)
+      mc.tasks.observeNodeField(task, "state", mc.tasks.onTaskStateChange, "both")
```

Comment:
let's have it so that if isTrackingTaskUsage is true, that we add the observer, and do nothing else yet.